docs/madr/decisions/073-spiffe-compliance.md

```diff
@@ -242,45 +245,55 @@ metadata:
 spec:
   selector:
     dataplane:
-      matchLabels:
-        app: test
+      matchLabels: # move to common structure (golang api)
+        app: test 
   spiffeID: # optional
-    trustDomain: "default.zone-1.0c5afd2e-de93-437b-8ba0-3decdff805b6.kuma.io" # { .Mesh }.{ .Zone }.{ .ClusterID }.kuma.io
+    trustDomain: "default.zone-1.kuma.io" # { .Mesh }.{ .Zone }.kuma.io
     path: "/ns/{{ .Namespace }}/sa/{{ .ServiceAccount }}" # let's use the same templating format as HostnameGenerator
   provider:
-    type: Provided | Spire
+    type: Bundled | Spire
     spire:
       agent:
         timeout: 1s
-    provided: # to extend in KM
-      trustExtractionDisabled: false # allows to disable extraction of CA into MeshTrust
-      insecureAutogenerate: false
-      certificate:
-        # one of
-        secret:
-        path:
-        envVar:
-      privateKey:
-        # one of
-        secret:
-        path:
-        envVar:
-      dataplaneCertificate:
-        duration: 24h
+    bundled: # to extend in KM
+      meshTrustCreation: Disabled | Enabled # allows to disable extraction of CA into MeshTrust
+      insecureAllowSelfSigned: true
```

Comment:
Does it make sense to do an enum? 🤔 

`insecureSelfSigned: allowed | disallowed`